ssl/t1_lib.c

```diff
@@ -1594,6 +1594,66 @@ int ssl_setup_sigalgs(SSL_CTX *ctx)
     return ret;
 }
 
+#define SIGLEN_BUF_INCREMENT 100
+
+char *SSL_get_builtin_sigalgs(OSSL_LIB_CTX *libctx)
```

Comment:
Given the function returns an allocated memory that needs to be freed, please rename it to SSL_get1_builtin_sigalgs().

Reply:
done